### PR TITLE
TeamZedbyte.testdashdrop version 0.0.1

### PR DESCRIPTION
**Pull request by arakesh7**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107752)

### CODE REVIEW

File: manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TeamZedbyte.testdashdrop
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Team Zedbyte
+PublisherUrl: https://github.com/ZEDBYTE-CROP/dashdrop-public/releases/download/v0.0.1/dashdrop.Setup.0.0.1.exe
+PackageName: testdashdrop
```
Comment (Trenly) [inline manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml:9]:
```suggestion
PackageName: dashdrop
```

File: manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TeamZedbyte.testdashdrop
+PackageVersion: 0.0.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/ZEDBYTE-CROP/dashdrop-public/releases/download/v0.0.1/dashdrop.Setup.0.0.1.exe
+  InstallerSha256: 4EF5A7904A2A8302334D4806359AEAC8FCC274B2DF12A16250359209E2A768AA
```
Comment (Trenly) [inline manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.installer.yaml:10]:
```suggestion
  InstallerSha256: 4EF5A7904A2A8302334D4806359AEAC8FCC274B2DF12A16250359209E2A768AA
  AppsAndFeaturesEntries:
  - DisplayName: dashdrop-uninstaller
```

File: manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TeamZedbyte.testdashdrop
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Team Zedbyte
+PublisherUrl: https://github.com/ZEDBYTE-CROP/dashdrop-public/releases/download/v0.0.1/dashdrop.Setup.0.0.1.exe
+PackageName: testdashdrop
+License: mit
```
Comment (Trenly) [inline manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml:10]:
```suggestion
License: MIT
```

File: manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: TeamZedbyte.testdashdrop
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Team Zedbyte
+PublisherUrl: https://github.com/ZEDBYTE-CROP/dashdrop-public/releases/download/v0.0.1/dashdrop.Setup.0.0.1.exe
+PackageName: testdashdrop
+License: mit
+Copyright: Copyright © 2023 Team Zedbyte
+ShortDescription: Dashdrop is a file transfer application developed by Zedbyte that allows users to transfer large file and folder containing file of up to 5TB quickly and easily.
+ReleaseNotes: this is release
```
Comment (Trenly) [inline manifests/t/TeamZedbyte/testdashdrop/0.0.1/TeamZedbyte.testdashdrop.locale.en-US.yaml:13]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**